ajax/libs/tailwindcss/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tailwindcss",
+  "filename": "tailwind.min.css",
+  "version": "0.4.1",
+  "description": "A utility-first CSS framework for rapidly building custom user interfaces.",
+  "license": "MIT",
+  "keywords": [
+    "tailwindcss",
+    "css",
+    "css-framework",
+    "postcss",
+    "functional-css",
+    "utility-classes",
+    "responsive"
+  ],
+  "author": "Adam Wathan",
```

Comment:
Let's use contributors info from the upstream
https://github.com/tailwindcss/tailwindcss/blob/master/package.json#L13-L17

Reply:
I use  
```
 "authors": [
    "Adam Wathan <adam.wathan@gmail.com>",
    "Jonathan Reinink <jonathan@reinink.ca>",
    "David Hemphill <davidlee.hemphill@gmail.com>"
  ],
```